content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -9,10 +9,11 @@ test: n/a
 ---
 
 This page describes the built-in configuration profiles that can be used when
-[installing Istio](/docs/setup/install/istioctl/).
-The profiles provide customization of the Istio control plane and of the sidecars for the Istio data plane.
-You can start with one of Istio’s built-in configuration profiles and then further customize the configuration for
-your specific needs. The following built-in configuration profiles are currently available:
+[installing Istio](/docs/setup/install/istioctl/). The profiles provide
+customization of the Istio control plane and of the sidecars for the Istio data
+plane. You can start with one of Istio’s built-in configuration profiles and
+then further customize the configuration for your specific needs. The following
+built-in configuration profiles are currently available:
 
 1. **default**: enables components according to the default settings of the
```

Comment:
Should we mention that the default profile creates a primary cluster?

Reply:
That is outside the scope of this PR.

Reply:
You're introducing primary cluster in this PR, hence it is in scope.